### DISCUSSION TITLE
Taxes

### DISCUSSION THREAD

**Discussion by CorwinDev** [Paymenter/Paymenter#172] [Ideas]
Being able to configure taxes.

It should be configurable if its INCLUDED in the product price or EXCLUDED

**Comment by Griffeng** [comment]
Display the tax on the invoice would also be nice so the customer knows what amount of tax they are paying. [This is an example from how WHMCS is displaying it.](https://i.imgur.com/822BpAL.png)

Some in-depth tax settings would also be nice like:

- Tax rules so we can setup tax for each country - A way to set a global tax rate that covers all country's as that is something applicable in some places 
- Ability to allow or disallow costumers to add there own VAT number to get tax exemption 

In general i would say just implement the settings WHMCS has as that covers most of the things required.

**Comment by CorwinDev** [comment]
Feature added in 0.9
